apps/arweave/src/ar_block_cache.erl

```diff
@@ -21,10 +22,10 @@
 %% validated: block is validated but does not belong to the tip fork
 %% not_validated: block is not validated yet
 %% none: null status
--type validation_status() :: on_chain | validated | not_validated | none.
--type nonce_limiter_validation_status() :: nonce_limiter_validated | 
-	nonce_limiter_validation_scheduled | awaiting_nonce_limiter_validation.
--type block_status() :: validation_status() | {not_validated, nonce_limiter_validation_status()}.
+%-type validation_status() :: on_chain | validated | not_validated | none.
+%-type nonce_limiter_validation_status() :: nonce_limiter_validated |
+%	nonce_limiter_validation_scheduled | awaiting_nonce_limiter_validation.
+%-type block_status() :: validation_status() | {not_validated, nonce_limiter_validation_status()}.
```

Comment:
you can just delete those if you want. I'd added them when I was trying to undesr the ar_block_cache for the checkpoint work. Probably not useful as a one-off